pkg/grpc/server_test.go

```diff
@@ -94,7 +94,17 @@ func TestClose(t *testing.T) {
 	t.Run("test close with api logging enabled", func(t *testing.T) {
 		port, err := freeport.GetFreePort()
 		require.NoError(t, err)
-		serverConfig := NewServerConfig("test", "127.0.0.1", port, []string{"127.0.0.1"}, "test", "test", 4, "", 4, true)
+		serverConfig := ServerConfig{
+			AppID:              "test",
+			HostAddress:        "127.0.0.1",
+			Port:               port,
+			APIListenAddresses: []string{"127.0.0.1"},
+			NameSpace:          "test",
+			TrustDomain:        "test",
+			MaxRequestBodySize: 4,
+			ReadBufferSize:     4,
```

Comment:
While you're at it, should we include the metric in the option name? `ReadBufferSizeKB/GB/MB/TB....`? 😄

Reply:
Clearly my sort algorithm for SI units is horribly broken...

Reply:
Sure I can make that change

Reply:
Changed